imports/ui/pages/currencyDetail/currencyInfo.js

```diff
@@ -18,11 +18,16 @@ Template.currencyInfo.onCreated(function () {
   })
   this.newAlgo = new ReactiveVar(false)
   this.showText = new ReactiveVar(false)
-
-  //binding for typeahead
   this.currency = new ReactiveVar(Template.currentData())
+
   this.autorun(()=>{
-    this.currency.set(Template.currentData())
+    this.currency = Template.currentData()
+    //for typeahead
+    if(Template.currentData().exchanges){
+      this.currency.exchangesNames = this.currency.exchanges.map(x=>{
```

Comment:
FYI The exchanges to list in the typeahead should be the exchanges that are *not* already associated with this currency.